op-chain-ops/genesis/layer_one.go

```diff
@@ -95,6 +97,17 @@ func BuildL1DeveloperGenesis(config *DeployConfig, dump *gstate.Dump, l1Deployme
 		}
 	}
 
+	beaconDepositAddr := common.HexToAddress("0x1111111111111111111111111111111111111111")
+	if err := beacondeposit.InsertEmptyBeaconDepositContract(memDB, beaconDepositAddr); err != nil {
+		return nil, fmt.Errorf("failed to insert beacon deposit contract into L1 dev genesis: %w", err)
+	}
+
+	// For 4788, make sure the 4788 beacon-roots contract is there.
+	// (required to be there before L1 Dencun activation)
+	memDB.CreateAccount(predeploys.EIP4788ContractAddr)
+	memDB.SetNonce(predeploys.EIP4788ContractAddr, 1)
+	memDB.SetCode(predeploys.EIP4788ContractAddr, predeploys.EIP4788ContractCode)
```

Comment:
For consistency with actual L1 we should also set the nonce on the EIP4788 sender account, otherwise it could run in a weird failed-tx case where the destination contract address already exists.